test/src/main/java/org/apache/accumulo/test/fate/FateIT.java

```diff
@@ -184,7 +167,7 @@ protected void testCancelWhileNew(FateStore<TestEnv> store, ServerContext sctx)
       // nothing should have run
       assertEquals(1, callStarted.getCount());
       fate.delete(txid);
-      assertThrows(getNoTxExistsException(), () -> getTxStatus(sctx, txid));
+      assertEquals(UNKNOWN, getTxStatus(sctx, txid));
```

Comment:
I'm glad you fixed this issue, I made the comment about the race condition because I encountered the same bug and I forgot to make a follow on issue to fix the test because it kept failing for me on occasion too